specification/communication/data-plane/SipRouting/preview/2021-05-01-preview/communicationservicessiprouting.json

```diff
@@ -112,6 +112,59 @@
           }
         }
       }
+    },
+    "/sip/checkRouting/{targetNumber}": {
```

Comment:
I'd recommend that this be a verb: `/sip:checkRouting` and that `targetNumber` be a required query parameter.

Justification:
* `/sip/checkRouting/{targetNumber}` suggests that the number being checked is a REST resource with CRUD operations. But that is not true. There is no resource backing this query, you can't create / update / delete this object.
* The `checkRouting` verb path infix shows that this is really an operation, not a resource (path parts should be nouns, not verbs).

Reply:
Used the verb notation and moved the parameter from path to query.